data/assets.yaml

```diff
@@ -119,102 +85,73 @@ shapefiles:
-  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_populated_places.zip
-  #   prj: 4326
-  #
-  # - name: ne_110m_admin_0_boundary_lines_land
-  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/ne_110m_admin_0_boundary_lines_land.zip
-  #   prj: 4326
-  #
-  # - name: ne_50m_admin_0_boundary_lines_land
-  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_0_boundary_lines_land.zip
-  #   prj: 4326
-  #
-  #- name: ne_10m_admin_0_boundary_lines_land
-  #  url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_land.zip
-  #  prj: 4326
-  #
-  #- name: ne_10m_admin_0_boundary_lines_map_units
-  #  url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_map_units.zip
-  #  prj: 4326
-  #
-  # - name: ne_50m_admin_1_states_provinces_lines
-  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_1_states_provinces_lines.zip
-  #   prj: 4326
-  #   shapefile-name: ne_50m_admin_1_states_provinces_lines.shp
-
-  # - name: ne_10m_admin_1_states_provinces_lines
-  #   url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_1_states_provinces_lines.zip
-  #   prj: 4326
-
   - name: ne_10m_populated_places
-    shapefile-name: ne_10m_populated_places.shp
-    url: *ne_pre_release_url
+    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_populated_places.zip
     prj: 4326
-    junk_dirs: yes
 
   - name: ne_110m_admin_0_boundary_lines_land
-    shapefile-name: ne_110m_admin_0_boundary_lines_land.shp
-    url: *ne_pre_release_url
+    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/110m/cultural/ne_110m_admin_0_boundary_lines_land.zip
     prj: 4326
-    junk_dirs: yes
 
   - name: ne_50m_admin_0_boundary_lines_land
-    shapefile-name: ne_50m_admin_0_boundary_lines_land.shp
-    url: *ne_pre_release_url
+    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_0_boundary_lines_land.zip
+    prj: 4326
+
+  - name: ne_10m_admin_0_boundary_lines_land
+    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_land.zip
+    prj: 4326
+
+  - name: ne_10m_admin_0_boundary_lines_map_units
+    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_boundary_lines_map_units.zip
+    prj: 4326
+
+  - name: ne_50m_admin_1_states_provinces_lines
+    url: http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_1_states_provinces_lines.zip
+    prj: 4326
+    shapefile-name: ne_50m_admin_1_states_provinces_lines.shp
```

Comment:
I think this is only necessary when there are a multiple shapefiles in a ZIP. It may be optional / ignored otherwise?